pkg/provider/azure_storageaccount.go

```diff
@@ -353,7 +353,11 @@ func (az *Cloud) createPrivateEndpoint(ctx context.Context, accountName string,
 		Location:                  &location,
 		PrivateEndpointProperties: &network.PrivateEndpointProperties{Subnet: &subnet, PrivateLinkServiceConnections: &privateLinkServiceConnections},
 	}
-	return az.privateendpointclient.CreateOrUpdate(ctx, vnetResourceGroup, privateEndpointName, privateEndpoint, true)
+
+	if rerr := az.privateendpointclient.CreateOrUpdate(ctx, vnetResourceGroup, privateEndpointName, privateEndpoint, "", true); rerr != nil {
```

Comment:
Why not just return `az.privateendpointclient.CreateOrUpdate(ctx, vnetResourceGroup, privateEndpointName, privateEndpoint, "", true).Error()`, like what you did on L387?

Reply:
can do that, actually